modules/resource-logs-viewing-cli-console.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * logging/resource-logs-viewing.adoc
+
+[id="resource-logs-viewing-cli-console_{context}"]
+= Viewing resource logs
+
+You can view the log for various resources in the {product-title} CLI and web console. Logs read from the tail, or end, of the log.
+
+.Prerequisites
+
+* Access to the OpenShift CLI (oc).
+
+.Procedure (UI)
+
+. In the {product-title} console, navigate to *Workloads* -> *Pods* or navigate to the pod through the resource you want to investigate. 
++
+[NOTE]
+====
+Some resources, such as builds, do not have pods to query directly. In such instances, you can locate the Logs link on the Details page for the resource.
+====
```

Comment:
Looks like the bolded `*Logs*` and `*Details*` might have been dropped in the last update.

Reply:
@lbarbeevargas Saw that. Neglected to push the change. Apologies!